libsplinter/src/rest_api/auth/actix.rs

```diff
@@ -18,7 +18,7 @@ use actix_web::dev::*;
 use actix_web::{
     http::{
         header::{self, HeaderValue},
-        Method,
+        Method as ActixMethod,
```

Comment:
This is more accurately `HttpMethod`

Reply:
The other `Method` enum imported in this file also represents HTTP methods. The key distinction is that one is provided by the Actix crate, the other is our own.